### PR TITLE
prometheus service metrics

### PR DESCRIPTION
**Pull request by murali-reddy**
prometheus service metrics

### CODE REVIEW

**Comment by murali-reddy** [comment]
`cloudnativelabs/kube-router-git:PR255` if any one want to give it a try.

**Comment by bzub** [comment]
Looks like it conforms to the format of [Prometheus adapter for custom metrics](https://github.com/DirectXMan12/k8s-prometheus-adapter/blob/master/docs/format.md). Great!

I will try to give it a spin later if no one else can today.

File: app/controllers/network_services_controller.go [hunk 1]
```diff
@@ -44,27 +44,22 @@ const (
...
-		Name:      "service_backend_pps_in",
+		Name:      "service_pps_in",
 		Help:      "Incoming packets per second",
-	}, []string{"namespace", "service_name", "backend"})
-	serviceBackendPpsOut = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	}, []string{"namespace", "service_name"})
+	servicePpsOut = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Namespace: namespace,
-		Name:      "service_backend_pps_out",
+		Name:      "service__pps_out",
```
Comment (tamalsaha) [inline app/controllers/network_services_controller.go]: is double underscore `__` intended ?

**Comment by murali-reddy** [comment]
Thanks @tamalsaha it was typo.

**Pull request merged**